packages/desktop/electron/electronApi.js

```diff
@@ -335,6 +335,12 @@ const ElectronApi = {
     isFeatureFlagEnabled(keyPath) {
         return keyPath?.split('.').reduce((prev, cur) => prev && prev[cur], features)?.enabled ?? false
     },
+    startLedgerProcess() {
+        return ipcRenderer.invoke('start-ledger-process')
```

Comment:
In cases where you're not waiting for the main process to return a response, it's better to use `ipcRenderer.send(...)` that is paired with an `ipcMain.on(...)`. 

Ref: https://www.electronjs.org/docs/latest/tutorial/ipc#pattern-1-renderer-to-main-one-way